lib/chef/provisioning/docker_driver/driver.rb

```diff
@@ -158,13 +159,23 @@ def destroy_machine(action_handler, machine_spec, machine_options)
         Chef::Log.debug('Already stopped!')
       end
 
-      Chef::Log.debug("Removing #{container_name}")
-      container.delete
-
-      Chef::Log.debug("Destroying image: chef:#{container_name}")
-      image = Docker::Image.get("chef:#{container_name}")
-      image.delete
+      begin
+      	Chef::Log.debug("Removing #{container_name}")
+      	container.delete	
+      rescue Docker::Error::ServerError
```

Comment:
Do we really want to destroy any containers when we destroy the image? The Docker CLI tool requires `-f` to delete images used by live containers.